### PR TITLE
New website design and texts

### PR DESCRIPTION
**Pull request by marcduiker**
TODO

- [x] Add latest hero image
- [x] Add images for the 4 value props
- [x] Update value prop texts
- [x] Add latest case studies to testimonials page
- [x] Add Enterprise page with links to Azure Container Apps and Diagrid Conductor.
- [x] Add more end user logos

### CODE REVIEW

File: themes/bigspring/layouts/partials/home/adopters.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{{ if site.Data.homepage.adopters.enable }}
+{{ with site.Data.homepage.adopters }}
+<section class="width-80 vertical-align-center padding-top-down-50">
+    <h2 class="text-center padding-top-down-10">Trusted and Adopted by</h2>
+    <div id="adopters" class="flex column-mobile">
+    {{ range .logos }}
+      <div class="img-container-md vertical-align-center">
```
Comment (yaron2) [inline themes/bigspring/layouts/partials/home/adopters.html:7]: For end user logos, please use: NASA, Bosch, IBM, Zeiss, ZScaler, Alibaba Cloud, Rakuten, PwC, Fujitsu, Vonage

Comment (yaron2) [reply themes/bigspring/layouts/partials/home/adopters.html:7]: Can also add Lufthansa @marcduiker

File: content/enterprise/_index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      link : "https://azure.microsoft.com/en-us/products/container-apps/"
+
+  - logo : "images/enterprise/diagrid.png"
+    alt: "Diagrid Conductor"
+    title: "Diagrid Conductor"
+    summary: "Let Conductor manage Dapr on your Kubernetes cluster so you can focus on writing code. Turn your Kubernetes cluster into a microservices powerhouse. Drastically reduce Dapr operational overhead while increasing reliability and security. Gain unparalleled visibility into your applications and infrastructure using the telemetry data generated by Dapr. Conductor tracks the health of your entire system and immediately shows you when something is wrong with built-in alerting for every Dapr metric."
+    cta :
+      enable : true
+      label : "Visit website"
+      link : "https://www.diagrid.io/"
```
Comment (yaron2) [inline content/enterprise/_index.md:25]: Change to `https://www.diagrid.io/conductor`.

File: content/enterprise/_index.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    cta :
+      enable : true
+      label : "Visit website"
+      link : "https://azure.microsoft.com/en-us/products/container-apps/"
+
+  - logo : "images/enterprise/diagrid.png"
+    alt: "Diagrid Conductor"
+    title: "Diagrid Conductor"
+    summary: "Let Conductor manage Dapr on your Kubernetes cluster so you can focus on writing code. Turn your Kubernetes cluster into a microservices powerhouse. Drastically reduce Dapr operational overhead while increasing reliability and security. Gain unparalleled visibility into your applications and infrastructure using the telemetry data generated by Dapr. Conductor tracks the health of your entire system and immediately shows you when something is wrong with built-in alerting for every Dapr metric."
```
Comment (yaron2) [inline content/enterprise/_index.md:21]: Change to: `Fully automated Dapr on any Kubernetes cluster with open-source enterprise support. Drastically reduce Dapr operational overhead while increasing reliability and security. Gain unparalleled visibility into your applications and infrastructure using the telemetry data generated by Dapr. Conductor tracks the health of your entire system and immediately shows you when something is wrong with built-in alerting for every Dapr metric.`

**Review by yaron2 — CHANGES_REQUESTED** [review]

File: data/homepage.yml [hunk 4]
```diff
@@ -1,14 +1,42 @@
...
-    link : "https://docs.dapr.io/getting-started/"
+####################### Case study #########################
+caseStudy:
+  enable : true
+  stories:
+    - name : "HDFC Bank"
+      title: "Handling millions of transactions efficiently with Dapr."
+      image : "images/casestudy/case-study-hdfc.png"
+      content :
+      link : "https://www.cncf.io/case-studies/at-bay/"
```
Comment (alicejgibbons) [inline data/homepage.yml:9]:
```suggestion
      link : "https://www.cncf.io/case-studies/hdfc-bank/"
```

File: data/homepage.yml [hunk 5]
```diff
@@ -1,14 +1,42 @@
...
+
+    - name : "DeFacto"
+      title: "How DeFacto migrated to an event-driven architecture with Dapr."
+      image : "images/casestudy/case-study-defacto.png"
+      content :
+      link : "https://www.cncf.io/case-studies/at-bay/"
```
Comment (alicejgibbons) [inline data/homepage.yml:15]:
```suggestion
      link : "https://www.cncf.io/case-studies/defacto/"
```

File: themes/bigspring/layouts/partials/home/cncf.html [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<div class="banner text-center">
+    <p>Dapr is a <a href="https://www.cncf.io/sandbox-projects/" class="banner-link">Cloud Native Computing Foundation incubation project</a></p>
```
Comment (alicejgibbons) [inline themes/bigspring/layouts/partials/home/cncf.html:2]:
```suggestion
    <p>Dapr is a <a href="https://www.cncf.io/projects/#incubating" class="banner-link">Cloud Native Computing Foundation Incubating project</a></p>
```

**Review by alicejgibbons — CHANGES_REQUESTED** [review]
Incorrect links

**Review by yaron2 — APPROVED** [review]

**Pull request merged**